### ISSUE TITLE
Add option to disable auto-roll damage

### ISSUE THREAD

**Issue by kakaroto** [kakaroto/Beyond20#11]
It's possible if I use 'rname' and 'rnamec' and make those into clickable links. They'll have to appear like the pink buttons in the spell descriptions, but it would work.
If I find a way to create macros I could call them with [rname](~macro_name), but I'm not sure I want to pollute macros or depend on being able to create macros.

**Comment by kakaroto** [comment]
```
c = window.Campaign.characters.get("beyond20")
if c is None:
   c = window.Campaign.characters.create({"name": "Beyond20 - Do not delete", "id": "beyond20")
c.abilities.create({"name": "Fireball - 1d6", "action":"&{template:simple} {etc..}", id="msg_hash"})
# Use [link](~beyond20|msg_hash)
```

**Comment by kakaroto** [comment]
nope, creating a character would only work if you're the DM, otherwise we get a permission denied.

**Comment by kakaroto** [comment]
Fixed in 04d8496b3f6045ae6c6f48ff4187e59b1acc2d79
